help/move-to-zulip-cloud.md

```diff
@@ -0,0 +1,135 @@
+# Move from self-hosting to Zulip Cloud
+
+With Zulip's high quality import and export tools, you can always
+move from self-hosting your own Zulip server to using the Zulip
+Cloud service (and back).
+
+## Process overview
+
+To move your Zulip organization from a self-hosted server to Zulip
+Cloud, you will need to take the following steps, which are described
+in more detail below:
+
+{start_tabs}
+
+1. [Plan the process and coordinate with Zulip support](#plan-the-process-and-coordinate-with-zulip-support).
+
+1. [Upgrade your self-hosted server](#upgrade-your-self-hosted-server).
+
+1. [Export organization data](#export-organization-data).
+
+1. [Decide how users will log in](#decide-how-users-will-log-in).
+
+{end_tabs}
+
+## Plan the process and coordinate with Zulip support
+
+To import your self-hosted organization into Zulip Cloud, your server will need
+to have the same database version as Zulip Cloud. Zulip Cloud is updated every
+couple of weeks, so it's important to coordinate the timing with Zulip's support
+team.
+
+{start_tabs}
+
+1. Review the process described on this page, and decide when you will be ready
+   to make the transition.
+
+1. [Create a new Zulip Cloud organization](https://zulip.com/new/)
+   with the subdomain you would like for your Zulip organization. Your
+   Zulip Cloud organization will be hosted at `<subdomain>.zulipchat.com`.
+
+1. Email [support@zulip.com](mailto:support@zulip.com) with the following information:
+      - URL of your new Zulip Cloud organization
+      - Your estimated timeline for generating a data export
+      - Any other timing considerations for the transition (e.g., time of day)
```

Comment:
Probably can add here:
- If you'd like any features that require the Plus plan, like [SAML authentication](https://zulip.com/help/saml-authentication) or a custom domain, details on how you'd like those to be set up.

Reply:
Wrote:

> If you're planning to purchase the [Zulip Cloud Plus plan](https://zulip.com/plans), details on the Plus plan features (e.g., authentication methods) you intend to use. These features will be configured for your organization as part of the import process.